src/Compilers/CSharp/Test/Emit/Emit/EmitErrorTests.cs

```diff
@@ -106,7 +106,11 @@ public static void Main()
     }
 }
 ";
-            VerifyEmitDiagnostics(source2, compilation1);
+            var compilation2 = CompileAndVerify(
+                source2,
+                new[] { new CSharpCompilationReference(compilation1) },
+                verify: false);
+            compilation2.Emit(expectedOutput: null, manifestResources: null, peVerify: false, expectedSignatures: null);
```

Comment:
@AlekseyTs CompileAndVerify doesn't go through the Emit phase where the error occurs.